src/TSHelper.ts

```diff
@@ -116,6 +116,15 @@ export class TSHelper {
         }
     }
 
+    public static getFunctionReturnType(node: ts.Node, checker: ts.TypeChecker): ts.Type {
```

Comment:
Maybe something like `getContainingFunctionReturnType` would be clearer

Reply:
Works for me.